mne/evoked.py

```diff
@@ -727,6 +727,47 @@ def plot_white(self, noise_cov, show=True):
         return _plot_evoked_white(self, noise_cov=noise_cov, scalings=None,
                                   rank=None, show=show)
 
+    def plot_joint(evoked, title='', picks=None, exclude=list(), show=True,
+                   ts_args=dict(spatial_colors=True), topomap_args=dict()):
```

Comment:
same thing about mutable defaults here. In the docstring you can say that `ts_args=None` will be translated to `dict(spatial_colors=True)`